### DISCUSSION TITLE
2024-09-20 온라인 회의

### DISCUSSION THREAD

**Discussion by nyj001012** [jiphyeonjeon-42/backend-nest#54] [General]
참여자 : yena, minsepar, youkim, jimin

지난 시간 목표설정 중간점검
이슈 공유

**Comment by nyj001012** [comment]
## 회의 일정 변경
- 주중을 온라인, 주말을 오프라인으로 하자

## 명세 작성
- `@anatine/zod-nestjs`로 변경

**Comment by scarf005** [comment]
## 작업 내용

- #50 
- #48
- #51 
- #53

**Comment by jimin52** [comment]
## @jimin52 에게 이슈 공유
- 명세 정리는 주말로 하기로 함
- 현재 api endpoint 옮기는 작업 중
    - 현재 서비스 백엔드에서 옮기는 중
- 온라인 회의를 주중에 하고 오프라인 회의를 주말에 하자는 의견 있었음

**Comment by jimin52** [comment]
오프라인 = 매주 토요일 오후 2시
온라인 = 매주 화요일 오후 9시

**Comment by jimin52** [comment]
문서화의 의미
- 개발환경 설정, 배포 방법 문서화
- 명세쪽 문서화

**Reply by jimin52** [reply]
다음주 토요일 회의에 길게 잡아서 명세쪽 문서화

**Comment by jimin52** [comment]
## 작업해야 하는 endpoint
- book (@jimin52  )
- user (@parkj12b)
- reservation
- auth (@scarf005 )
- cursus (@nyj001012 )
- search_keyword
- stock

**Comment by jimin52** [comment]
minsepar 님 질문
request 쪽에서 param 받을 때 zod validation 진행 할텐데 테이블에 따라서 string 으로 되어있을 때
테이블에 column type 이 varchar(255) 인 경우 client 의 입력을 제한해야 할지 궁금했음
답변
최대한 앞단에서 (zod 쪽에서) 자르는 게 좋을 것 같습니다.
